src/topp/HighResPrecursorMassCorrector.cpp

```diff
@@ -123,7 +124,12 @@ class TOPPHiResPrecursorMassCorrector :
       registerStringOption_("nearest_peak:mz_tolerance_unit", "<choice>", "ppm", "Unit of precursor mass tolerance", false);
       setValidStrings_("nearest_peak:mz_tolerance_unit", ListUtils::create<String>("Da,ppm"));
 
-      registerOutputFile_("out_csv", "<file>", "", "Optional CSV output file for results on 'nearest_peak' algorithm (see corresponding subsection) containing columns: " + ListUtils::concatenate(ListUtils::create<String>(csv_header), ", ") + ".", false);
+      registerTOPPSubsection_("highest_intensity_peak", "Use centroided MS1 peak with the highest intensity in a certrain mass range - for precursor mass correction");
+      registerDoubleOption_("highest_intensity_peak:mz_tolerance","<num>", 0.0,"The precursor mass tolerance to find the highest intensity MS1 peak (Da) (Use 0.2 Da as default)", false);
```

Comment:
The problem is, that later (line 544) it will be asked 
` if (highest_intensity_peak_mz_tolerance > 0.0)
      {
        corrected_to_highest_intensity_peak = correctToHighestIntensityMS1Peak(exp, highest_intensity_peak_mz_tolerance, deltaMZs, mzs, rts);
      }`
Therefore the default value should be 0.0 and then set to 0.2 if the highest_intensity_peak method should be used in the analysis or would you suggest another approach.
The same is true for the nearest_peak method.